cli.py

```diff
@@ -42,7 +43,13 @@ def update_normalizer_db(normalizer, dev, db_url, update_all):
         elif db_url:
             db: Database = Database(db_url=db_url)
         else:
-            db: Database = Database()
+            check = input("Enter 'yes' if you want to use production db: ")
+            if check.lower() == 'yes':
+                click.echo("Updating production db...")
+                db: Database = Database()
+            else:
+                click.echo("Exiting CLI.")
+                sys.exit()
```

Comment:
What about something like this?  https://click.palletsprojects.com/en/5.x/prompts/#confirmation-prompts

```suggestion
            if click.confirm("Are you sure you want to update the production database?", default=False):
                click.echo("Updating production db...")
                db: Database = Database()
            else:
                click.echo("Exiting CLI.")
                sys.exit()
```